packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css

```diff
@@ -0,0 +1,44 @@
+.root {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space--12);
+  }
+  
+ /*  .card {
+  height: 470px;
+  width: 248px;
+  background-color: #191B38;
+  gap: 16px;
+  border-radius: var(--border-radius--8);
+  }
+ */
+  .imageWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: var(--border-radius--8);
+    overflow: hidden;
+    color: var(--color-purple--9);
+    background-color: #191B38;
+    transition: 0.2s;
+    aspect-ratio: 3 / 4;
+  }
+  
+
+  .title {
+    overflow: hidden;
+    color: var(--color-text--default);
+    font-weight: var(--font-weight-bold);
+    font-size: var(--font-size--l);
+    line-height: normal;
+    white-space: nowrap;
+    text-overflow: ellipsis;

```

Comment:
These are not needed and there should only be the box height and width from the designs. (width being the 80%, we talked about, oslt) Then the border radius and probably the color.